### ISSUE TITLE
Tinker Construct mod compatible Issue

### ISSUE THREAD

**Issue by ClockworkzAI** [Yesssssman/epicfightmod#809]
https://www.curseforge.com/minecraft/mc-mods/tinkers-construct

#549
Its still a issue in 16.5.11 don't know if it is still a issue with the newer version.

**Comment by Bitgedon** [comment]
Considering how tinkers construct was and still is one of the most complicated weapons system mods to date. I doubt it will ever become fully compatible. I remember someone having made a config somewhere to get the animations working but the weapon effects do not.

**Comment by ClockworkzAI** [comment]
Yes i already set up the animation for the tools and weapons btw did you just joined yesterday to tell me this or am i wrong?

**Comment by DarkScorn** [comment]
It's still an issue 16.5.13. Severing modifier applies in combat mode fight, but many others (bleeding, insatiable, piercing, etc) do not apply at all. It might be related to the other "Fully charged attack" issue.
